src/main/kotlin/mynameisjeff/skyblockclientupdater/utils/UpdateChecker.kt

```diff
@@ -163,8 +163,8 @@ object UpdateChecker {
 
         val e = expected.toCharArray().dropWhile { it == '!' }.filter { !exempt.contains(it) }
         val r = received.toCharArray().dropWhile { it == '!' }.filter { !exempt.contains(it) }
-        val distance = StringUtils.getLevenshteinDistance(expected, received)
-        if (distance !in 1..6) return false
+        val distance = StringUtils.getLevenshteinDistance(e, r)
```

Comment:
this is 2 char arrays, not 2 strings

you need to convert them to strings first